opentelemetry-stdout/src/trace/exporter.rs

```diff
@@ -107,6 +117,7 @@ impl SpanExporterBuilder {
     pub fn build(self) -> SpanExporter {
         SpanExporter {
             writer: Some(self.writer.unwrap_or_else(|| Box::new(stdout()))),
+            resource: Resource::default(),
```

Comment:
could we do empty here instead, as default is not really empty, and will unnecessarily waste cycles to populate the default, only to throw it away and override with SetResource.